scvid/train/training_plan.py

```diff
@@ -0,0 +1,51 @@
+from typing import Optional
+
+import pyro
+import pytorch_lightning as pl
+import torch
+
+
+class PyroTrainingPlan(pl.LightningModule):
```

Comment:
Might want to add to the Docstring that this implementation is a stripped-down version of what is in scvi-tools and also provide a permalink reference.